### PR TITLE
Parse and show date of collection

### PR DESCRIPTION
**Pull request by justinvdk**
This PR just checks the content of the cell before assuming it is a date.

### CODE REVIEW

**Comment by wouterjsmit** [comment]
Not needed for the demo right? Assigning to next sprint then

**Comment by justinvdk** [comment]
You're right. I thought I did? Or maybe no milestone at all. Next week is fine

**Comment by FarisElghlan** [comment]
I guess this can be merged? Code looks good at least, so imo no reason to wait.

File: PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/model/Annotation.java [hunk 1]
```diff
@@ -120,7 +121,12 @@ public String buildMetaDataString() {
     sb.append("Age\t\t\t\t\t\t").append(age).append('\n');
     sb.append("Capreomycin\t\t\t\t").append(capreomycin).append('\n');
     sb.append("Cohort\t\t\t\t\t").append(cohort).append('\n');
-    sb.append("Date of collection\t\t\t").append(dateOfCollection).append('\n');
+    sb.append("Date of collection\t\t\t");
+    if (dateOfCollection != null) {
```
Comment (wouterjsmit) [inline PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/model/Annotation.java]: Both branches are one line long. It is better practice/easier to read to read a positive if-condition than a negative. This is a [style rule imposed by PMD](http://pmd.sourceforge.net/pmd-4.3.0/rules/design.html#ConfusingTernary) (also in our project).

**Comment by wouterjsmit** [comment]
One comment. We all know it is pointless, but it is PMD and we need to also start fixing our site-generated PMD warnings (see #62)

**Comment by justinvdk** [comment]
@Pathemeous fixed it. I think for null-checks the positive version is more confusing because you usually ask yourself "when stuff's alright, go-on".

This is why Java needs to result non-null references to true... (Who cares about the Boolean class anyway ¯_(ツ)_/¯)

**Pull request merged**